rest/data/perf.go

```diff
@@ -281,7 +343,7 @@ func (mc *MockConnector) RemovePerformanceResultById(_ context.Context, id strin
 // performance results with the given taskId and that fall within interval,
 // filtered by tags.
 func (mc *MockConnector) FindPerformanceResultsByTaskId(_ context.Context, taskId string, interval util.TimeRange, tags ...string) ([]dataModel.APIPerformanceResult, error) {
-	results := []dataModel.APIPerformanceResult{}
+	var results []dataModel.APIPerformanceResult
```

Comment:
there's no reason to make this change, or ones like them below...

Reply:
IDE was complaining about it --> I think this marginally saves some memory that would otherwise be allocated: https://stackoverflow.com/questions/29164375/correct-way-to-initialize-empty-slice.

Reply:
I can't imagine that matters, particularly in the mock, and I would (with the possible exception of gofmt changes) avoid making changes outside of the scope of your ticket unless needed, just to make things easier to review, and also to make looking over diffs easier later.